e2e/test/e2e_test.go

```diff
@@ -302,3 +305,23 @@ func fetchBlockChain(ctx context.Context, t *testing.T) []*cmttypes.Block {
 
 	return blocks
 }
+
+func makePortalRegistry(network netconf.ID, endpoints xchain.RPCEndpoints) (*bindings.PortalRegistry, error) {
```

Comment:
note this helper function is duplicated in a few places. I'm not sure where to put a common one, since it depends on a bunch of different packages. Maybe a common `wire` package. But not sure it is worth it. So just duplicated it for now.

Reply:
Maybe we need a `portal` or `portalregistry` package?

Reply:
I'm not so sure.  A package with just a single function? Doesn't feel worth it.